node/api/server/server.go

```diff
@@ -88,3 +104,14 @@ func New(APIaddr string, requiredUserAgent string, requiredPassword string, node
 
 	return srv, nil
 }
+
+// MineBlock makes the underlying node mine a single block and broadcast it.
+func (srv *Server) MineBlock() error {
```

Comment:
This needs to be a function of the test node in the `siatest` package, the `Server` should be treated as a simplified object that we intend to use in production